### ISSUE TITLE
Adjustable size of the code?

### ISSUE THREAD

**Issue by MaxValue** [evgeni/qifi#13]
This is especially relevant for the print-layout.

This issue should be actually called "adjustable style", but i know this is somewhat of a bigger request. If at least the size would be changeable, that'd be cool.

**Comment by evgeni** [comment]
you mean you want the qr code printed in a different size? most browsers will allow you to adjust the scaling of the page before printing, but I admit this is kinda awkward.

**Comment by MaxValue** [comment]
Yeah, this is what I am doing in any case. Because my wifi password was rather long (16 characters), it cut off the last character a little bit on the printed out version, so its not that reliable.

What I actually meant was: independently from the font size or the paper size, make the code itself resizeable (either by defining the dot size or by defining the side length, but I think exposing the side length as a parameter for the user is more understandable).
